src/components/m-banner-message.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="m-banner-message bg--secondary text--white px-2 py-2">
```

Comment:
Tu peux utiliser que `m-banner-message` en class et extends les autres, ce sera moins éparpillé

```suggestion
  <div class="m-banner-message">
```